eds/blocks/tabs/tabs.css

```diff
@@ -161,6 +161,7 @@
       align-items: center;
       justify-content: center;
       margin-block-end: 0;
+      inline-size: 1420px;
```

Comment:
Should you set this to `inline-size: 100%;` instead since the parent `tab-nav` class already defines the width as `1420px`?

Reply:
Ah, good catch